src/marine/cryosat_ice2ioda.py

```diff
@@ -124,16 +118,25 @@ def main():
 
     args = parser.parse_args()
     fdate = datetime.strptime(args.date, '%Y%m%d%H')
-    writer = iconv.NcWriter(args.output, locationKeyList)
+    VarDims = {
+        vName: ['nlocs'],
+    }
 
     # Read in
-    ice = Observation(args.input, args.thin, fdate, writer)
+    ice = Observation(args.input, args.thin, fdate)
 
     # write them out
-    AttrData['date_time_string'] = fdate.strftime("%Y-%m-%dT%H:%M:%SZ")
 
-    (ObsVars, LocMdata, VarMdata) = writer.ExtractObsData(ice.data)
-    writer.BuildNetcdf(ObsVars, LocMdata, VarMdata, AttrData)
+    ObsVars, nlocs = iconv.ExtractObsData(ice.data, locationKeyList)
+    DimDict = {'nlocs': nlocs}
+    writer = iconv.IodaWriter(args.output, locationKeyList, DimDict)
+
+    VarAttrs = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
+    VarAttrs[vName, iconv.OvalName()]['units'] = 'm'
```

Comment:
What about fill values? Are they the standard ones? Also, it would be preferred if `VarAttrs` were added around line 43, but this is okay as is I think.

Reply:
I added fill values. The fill values are copied from the original Cryosat_ice netcdf file for Oval and Oqc. I added -999. for Oerr.  Thank you @Cory!